### DISCUSSION TITLE
Guardar certificado no bando de dados

### DISCUSSION THREAD

**Discussion by EduardoRBarbosa** [TadaSoftware/PyNFe#241] [General]
Necessito ler o certificado no banco de dados em nuvem,  alguem ja passou por isto ? Pois o PyNfe, le direto o arquivo, como o arquivo deve ficar em nuvem, estou com muita dificuldade neste passo. Obrigado

**Comment by rodrigoneal** [comment]
Você pode pegar os dados em nuvem e criar um arquivo .pfx quando for usar. Crie um arquivo temporário para ser excluído depois de utilizado.
